ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3960,6 +3960,16 @@ BYPDM:
   - MONITORING
 
 
+BYPDVM:
```

Comment:
@shambergoldstein No sensor field was seen in our data. Does modeling this require a sensor field?

Reply:
We generally want feedback with commands. I recommend adding a sensor and marking it missing in the building config.

Reply:
@shambergoldstein I have added bypass_air_damper_voltage_sensor as a required field as you suggested.